lib/scope.js

```diff
@@ -235,6 +235,7 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
                     ref.reference(options);
                 });
                 node.thedef = def;
+                node.reference(options);
```

Comment:
I'm assuming that `node` could literally reference any object or array here and still work?